tests/us_arboretum_test.py

```diff
@@ -0,0 +1,28 @@
+from os import path
+import sys
+import unittest
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from events.us_arboretum import main
+from .utils import schema_test_required, schema_test_all, schema_test_types
+
+class ANSTestCase(unittest.TestCase):
```

Comment:
Please rename test case to something like `UsArborteumTestCase` and update doc string.